index.html

```diff
@@ -438,6 +438,27 @@ <h2>Data Model</h2>
 <a>verification methods</a>.
       </p>
 
+      <p>
+All of the data model terms and types in this specification map to URLs. The
+vocabulary where these terms are defined is the [[?SECURITY-VOCABULARY]]. The
+explicit mechanism that is used to perform this mapping in a secured document
+is the `@context` property.
+      </p>
+      <p>
+When processing a document as JSON-LD, the mapping mechanism is defined by
+JSON-LD [[JSON-LD11]]. When processing a document in a non-JSON-LD environment,
+the same mechanism is used to ensure that the terms have the same interpretation
+regardless of format, but with a simpler processing rule: Implementers
+performing non-JSON-LD processing MUST ensure that all values associated with a
+`@context` property are in the expected order, the contents of the context files
+match known good cryptographic hashes for each file, and the contents are known
+to be valid. Implicit `@context` values MAY be used such that no `@context`
+value needs to be present in a non-JSON-LD document. Using static context files
```

Comment:
I am not sure what this "Implicit…document" sentence means, and I am sure that a non-JSON-LD implementer will understand even less that what I understand...

Reply:
It's a bit of a hand-wave right now... I was planning on writing some documentation for it in a future PR. Fundamentally, what it's trying to say is: "You can use Data Integrity w/ non-JSON-LD documents.":

https://w3c.github.io/vc-di-eddsa/#jcs-eddsa-2022

IOW, in a JSON-only environment, you can presume an implicit `@context` value (the data integrity context) that would define the data integrity only terms and leave all the other JSON-only terms as undefined. That is, of course, a slippery slope because the very next question one might have is "well, what do we do with the undefined JSON terms?" and the JSON answer to that is: Nothing, it's just JSON, you're going to have to figure out what those terms mean by talking to someone or reading some documentation somewhere on the Web. The JSON-LD answer to that is "well, the Data Integrity terms will be machine-readable, but the undefined JSON terms will throw undefined errors unless you also presume some base vocabulary, and that's up to your specification to do".

All of the above said, we should probably just remove that sentence until we added the appropriate section, or decide that the whole implicit `@context` thing is a bad idea, because it introduces more problems than just injecting an `@context` value (or depending on one to already exist, like the VC v2.0 context).

I'll remove the sentence in the next revision to this PR.

Reply:
I ended up trying to define it more accurately here: bbbec99f19a984ae54c342a5c548589dbfd449bd